netlogo-gui/src/main/app/InterfaceToolBar.scala

```diff
@@ -34,6 +34,18 @@ class InterfaceToolBar(wPanel: WidgetPanel,
   private val deleteAction = new DeleteAction()
   private val deleteButton = new ToolBarActionButton(deleteAction)
   private val widgetMenu = new WidgetMenu
+  private val undoButton = new JButton("Undo")
```

Comment:
This should subclass `ToolBarActionButton` or a related class so that it has the same appearance as the other buttons.